altairexpress/altairexpress.py

```diff
@@ -53,8 +52,44 @@ def ts_alt(data, col, type, frequency):
     Examples
     --------
     >>> from altairexpress import altairexpress
-    >>> altairexpress.ts_alt("https://raw.githubusercontent.com/plotly/datasets/master/timeseries.csv", A, irregular, 3)
+    >>> altairexpress.ts_alt("https://raw.githubusercontent.com/plotly/datasets/master/timeseries.csv", A, 3)
     altair.vegalite.v3.api.Chart
     """
+    # Check the variable type of inputs
+    assert isinstance(data, str), "TypeError: The path of the data must be entered as a string."
+    assert isinstance(col, str), "TypeError: The column name must be entered as a string."
+    if (frequency not in [1, 4, 12, 52]):
+        raise Exception("ValueError: Frequency must be an integer from {1, 4, 12, 52}.")
+    
+    # load the data
+    df = pd.read_csv(data, index_col=0, parse_dates=True)
+    df_res = df.reset_index()
```

Comment:
Can condense into one line

```suggestion
    df_res = pd.read_csv(data, index_col=0, parse_dates=True).reset_index()
   
```